kfs-core/src/main/java/edu/arizona/kfs/fp/document/CashReceiptDocument.java

```diff
@@ -0,0 +1,83 @@
+package edu.arizona.kfs.fp.document;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.fp.businessobject.Check;
```

Comment:
There are 23 unused imports here which should be removed (Eclipse hint: ctrl+shift+o will auto-organize and remove dead imports).